langchain4j-core/src/main/java/dev/langchain4j/data/audio/Audio.java

```diff
@@ -10,6 +10,7 @@
 @Experimental
 public class Audio {
     private final URI url;
+    private final byte[] audioData;
```

Comment:
```suggestion
    private final byte[] binaryData;
```
Not sure about this option either

Reply:
What do you think @langchain4j if we move this class into the azure openai lc4j module ? this object corresponds to Azure Open AI case

Reply:
@anunnakian The Audio class is generic, not Azure-specific:

It's in the [langchain4j-core](https://humble-dollop-vwr6q75gp72rv6.github.dev/) module under dev.langchain4j.data.audio
It's part of the core audio abstraction alongside AudioTranscriptionRequest and AudioTranscriptionResponse
The AudioModel interface in core depends on it
It's designed for reusability:

Any future audio model implementation (OpenAI, Google, AWS, etc.) would need this class
Moving it to Azure module would create circular dependencies or force duplication
Current architecture is correct:

Core abstractions (Audio, AudioModel, Request/Response) belong in [langchain4j-core]
Implementation-specific code belongs in provider modules (azure-open-ai, openai, etc.)